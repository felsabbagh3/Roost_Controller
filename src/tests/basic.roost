INIT_TEMP 70
0 SET_TEMP 60
0 PRINT_STATUS
10 PRINT_STATUS
50 PRINT_STATUS
100 PRINT_STATUS
100 SET_TEMP 80
200 PRINT_STATUS
