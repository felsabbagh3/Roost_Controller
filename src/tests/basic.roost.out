Roost Status =====================
| Time:            0
| Cur. Temp:       70
| Desired Temp:    65
| Current routine: (none) (0) 
==================================
Roost Status =====================
| Time:            10
| Cur. Temp:       65.7357
| Desired Temp:    65
| Current routine: (none) (10) 
==================================
Roost Status =====================
| Time:            50
| Cur. Temp:       65.2243
| Desired Temp:    65
| Current routine: (none) (50) 
==================================
Roost Status =====================
| Time:            100
| Cur. Temp:       64.8669
| Desired Temp:    65
| Current routine: (none) (100) 
==================================
Roost Status =====================
| Time:            200
| Cur. Temp:       78.9915
| Desired Temp:    80
| Current routine: (none) (200) 
==================================
Simulation complete. Total energy usage: 15.1
